cmd/ef_tests/test_runner.rs

```diff
@@ -5,27 +5,63 @@ use ethereum_rust_core::{
     rlp::decode::RLPDecode,
     rlp::encode::RLPEncode,
     types::{Account as CoreAccount, Block as CoreBlock, Transaction as CoreTransaction},
-    Address,
+    Address, U256,
 };
 use ethereum_rust_evm::{
-    apply_state_transitions, evm_state, execute_tx, process_withdrawals, EvmState, SpecId,
+    apply_state_transitions, beacon_root_contract_call, evm_state, execute_block, execute_tx,
+    process_withdrawals, EvmState, SpecId,
 };
 use ethereum_rust_storage::{EngineType, Store};
 
+/// Tests the execute_block function
+pub fn execute_test2(test: &TestUnit) {
+    // Build pre state
+    let mut evm_state = build_evm_state_from_prestate(&test.pre);
+    apply_state_transitions(&mut evm_state).expect("Failed to update DB state");
```

Comment:
Is this call needed? The state has just been created, there are no transitions to apply